allennlp/commands/make_vocab.py

```diff
@@ -63,35 +70,36 @@ def make_vocab_from_args(args: argparse.Namespace):
     """
     parameter_path = args.param_path
     overrides = args.overrides
+    serialization_dir = args.serialization_dir
 
     params = Params.from_file(parameter_path, overrides)
 
-    make_vocab_from_params(params)
+    make_vocab_from_params(params, serialization_dir)
 
-def make_vocab_from_params(params: Params):
+def make_vocab_from_params(params: Params, serialization_dir: str):
     prepare_environment(params)
 
     vocab_params = params.pop("vocabulary", {})
-    vocab_dir = vocab_params.pop('directory_path', None)
-    if vocab_dir is None:
-        raise ConfigurationError("To use `make-vocab` your configuration must contain a value "
-                                 "at vocabulary.directory_path")
+    os.makedirs(serialization_dir, exist_ok=True)
+    vocab_dir = os.path.join(serialization_dir, "vocabulary")
 
-    os.makedirs(vocab_dir, exist_ok=True)
+    if os.path.isdir(vocab_dir) and os.listdir(vocab_dir) is not None:
+        raise ConfigurationError("The 'vocabulary' directory in the provided"
```

Comment:
Extra space needed here at the end of one of the lines.

Reply:
Sure 👍